renovate.json

```
@@ -1,13 +1,11 @@
 {
-    "extends": [
-        "github>konflux-ci/mintmaker//config/renovate/renovate.json"
-    ],
-    "enabledManagers": [
-        "docker",
-        "go",
-        "kustomize"
-    ],
-    "go": {
-        "pinDigests": false
-    }
+    "gomod": {
+        "enabled": true,
+        "postUpdateOptions": ["gomodTidy"]
+    },
+    "kubernetes": {
+        "fileMatch": [
+            ".*.ya?ml$",
```

I think it's enough to match the extension in this case, but regardless, need to escape the dot.
Something like this should work:
```suggestion
            "\\.yaml$",
             "\\.yml$"
```

---

Fixed in b769244